src/java.base/share/classes/java/lang/classfile/Opcode.java

```diff
@@ -1118,4 +1122,1228 @@ public static enum Kind {
      * {@return instruction kind}
      */
     public Kind kind() { return kind; }
+
+    /**
+     * Holds the constant values of the Opcodes.
+     *
+     * @see Opcode#bytecode() Opcode::bytecode
+     * @since 24
+     */
+    @PreviewFeature(feature = PreviewFeature.Feature.CLASSFILE_API)
+    public static final class OpcodeValues {
```

Comment:
I think we should not introduce a new API class just to expose int we plan to hide under the Opcode.
My proposal is to hard-code the opcodes in the Opcode initialization and remove OpcodeValues class.

Reply:
I think we can move these values to an implementation class like `RawBytecodeHelper` as we make use of these values. We can always move these constants back if there is need.